File: client/real-estate-crm-react/src/types/company.ts
export interface Company {
  id: string;
  name: string;
  slug: string;
  phone: string | null;
  email: string | null;
  isActive: boolean;
}

export interface DashboardSummary {
  totalLeads: number;
  newLeadsLast30Days: number;
  conversionRatePercent: number;
  totalDeals: number;
  totalActiveDeals: number;
  totalSalesValue: number;
  upcomingFollowUps: number;
  totalAvailableUnits: number;
}

File: src/RealEstateCRM.Domain/Entities/Company.cs
using RealEstateCRM.Domain.Common;

namespace RealEstateCRM.Domain.Entities;

public class Company : BaseEntity
{
    public string Name { get; set; } = string.Empty;
    public string Slug { get; set; } = string.Empty;
    public string? Phone { get; set; }
    public string? Email { get; set; }
    public bool IsActive { get; set; } = true;
}
